return_interface_cones.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[ ]:
+
+
+def parse_cones_file(pathname, dictionary, switch):
+    '''Parses a .cones file and returns a formatted dictionary of all of its cones'''
+    pathname = pathname.strip() # eliminate embedded null character
+    if switch == "acceptorCones": # ACCEPTOR
+        with open(pathname, 'r') as f:
+            for line in f:
+                if line.startswith("CONES:"): #why do we skip lines starting with 'cones'? 
+                    break
+            for line in f:
+                coors = line.strip().split() # removes any leading and trailing whitespaces and split the resulting string
+                key = line.strip().split('[')[1][:-14] 
+                apex = tuple(coors[0:3])
+                perp_vec = tuple(coors [3:6])
+                dictionary['ACCEP ' + key] = [apex, perp_vec]
+    else: # DONOR 
+        with open(pathname, 'r') as f:
+            for line in f:
+                if line.startswith("CONES:"):
+                    break
+            for line in f:
+                coors = line.strip().split()
```

Comment:
Explain the meaning of the data stored in variables `coors`, `apex`, `perp_vec`, and `key`